src/app/page.tsx

```diff
@@ -2,7 +2,7 @@ import Image from "next/image";
 
 export default function Home() {
   const num = 1;
-  const str = "Hello, world!";
+  const str = "Hello, ReviewDog!";
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'str' is assigned a value but never used.
<!-- __reviewdog__:ChBmMDI4MzhmNGI5ZGYwMzM5EgZlc2xpbnQ= -->